recipes-ni/pstore-save/files/pstore-save

```diff
@@ -46,6 +46,7 @@ done
 
 function log_msg () {
     echo "$SCRIPTNAME: $@"
+    logger "$SCRIPTNAME: $@"
```

Comment:
Default log level is `user.notice`. I'm assuming that's sufficient.

Reply:
Does this actually do anything during boot? At `S09`, the `pstore-save` [initscript runs](https://github.com/ni/meta-nilrt/blob/4402ef086bb2d9b4fdfd47845dbcd0100dbfe211/recipes-ni/pstore-save/pstore-save_1.0.bb#L19) and [calls the `pstore-save` bin script](https://github.com/ni/meta-nilrt/blob/4402ef086bb2d9b4fdfd47845dbcd0100dbfe211/recipes-ni/pstore-save/files/initscript#L36), and redirects stdout to `/dev/ksmg`. So I think the `echo` line above here is what is getting logged to the kernel log.

The `syslog` daemon starts at `S20`, so I don't think the `logger` call would actually do anything during boot - which is the same issue that I had in #450, and why I had to split the initscripts for the `ni-rtfeatures` package.

Reply:
I see entries in both syslog and dmesg after manually crashing a target. Does `/dev/kmsg` get output to `/var/log/messages` as well?

Reply:
After some more poking around, I think this change is unnecessary. Since we're logging to the kernel ring buffer during boot, `syslogd` already grabs the log item and copies it to `/var/log/messages` when the daemon starts. Abandoning this PR.